packages/api/src/models/product-info/product-info.ts

```diff
@@ -6,7 +8,7 @@ export class ProductInfo {
   sesame: string;
   connectors: string;
 
-  constructor(data: Partial<ProductInfo & { Workbench: string }>) {
+  constructor(data: ProductInfoDto) {
```

Comment:
Seen throughout the MR.

Using `ProductInfoDto` in the model is not ok. This is what the mapper is for. To get the DTO as an argument and poop the instantiated object. Both models should not be coupled.